package.json

```diff
@@ -14,6 +14,7 @@
     "@vercel/build-utils": "*",
     "@vercel/ncc": "0.24.0",
     "@vercel/next": "*",
+    "@vercel/static-build": "*",
```

Comment:
Is this necessary?

Reply:
yes - same reason we had to put `@vercel/next` in here. Its used by the `test/lib/runBuildLambda.js`

Reply:
but i actually don't know if anything is actually running the tests in `packages/static-build/test/builds.test.js`

Reply:
In a separate PR we should refactor `runBuildLambda()` to accept an input `build()` function that would be imported from the builders' tests. That way these references at the root would not be necessary.

Reply:
See https://github.com/vercel/vercel/pull/9262.